action/steps/run_tests.sh

```diff
@@ -111,7 +111,7 @@ if [ $PLAY_MODE = true ]; then
   echo "#   Testing in PlayMode   #"
   echo "###########################"
   echo ""
-  unity-editor \
+  /opt/unity/Editor/Unity \
```

Comment:
We are using the alias on purpose - it has compatibility fixes and environment context.

Reply:
I understand. But the alias is using some commands that restrict me from using nvidia-docker.